blob/blob.go

```diff
@@ -26,8 +26,8 @@ import (
 	"github.com/google/go-cloud/blob/driver"
 )
 
-// Reader implements io.ReadCloser to read from blob. It must be closed after
-// reads finished. It also provides the size of the blob.
+// Reader implements io.ReadCloser to read a blob. It must be closed after
+// reads are finished. It also provides the Size and ContentType of the blob.
```

Comment:
Since ContentType and Size are methods and not fields in Reader, I suggest removing the last sentence.

Reply:
Done.